.env.example

```diff
@@ -2,6 +2,7 @@ DATABASE_ENGINE=django.db.backends.postgresql
 DATABASE_NAME=postgres
 DATABASE_USER=postgres
 DATABASE_HOST=db
+POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
Can we move this into the `docker-compose.yml`? It's nothing sensitive and needs to be in there for the dev env to work.

Reply:
We can but won't we have to move it again when we start using Docker on AWS?